src/wp-includes/block-template-utils.php

```diff
@@ -1315,7 +1340,11 @@ function wp_generate_block_templates_export_file() {
 	// Load templates into the zip file.
 	$templates = get_block_templates();
 	foreach ( $templates as $template ) {
-		$template->content = _remove_theme_attribute_in_block_template_content( $template->content );
+		$template_blocks   = parse_blocks( $template->content );
+		$template->content = traverse_and_serialize_blocks(
+			$template_blocks,
+			'_remove_theme_attribute_from_template_part_block'
+		)
```

Comment:
```suggestion
		$template->content = traverse_and_serialize_blocks(
			parse_blocks( $template->content ),
			'_remove_theme_attribute_from_template_part_block'
		)
```

Remove local variable